calendra/europe/united_kingdom.py

```diff
@@ -18,26 +17,72 @@ class UnitedKingdom(WesternCalendar, ChristianMixin):
     include_easter_monday = True
     include_boxing_day = True
     shift_new_years_day = True
+    non_computable_holiday_dict = {
+        1973: [(date(1973, 11, 14), "Royal wedding"), ],
+        1977: [(date(1977, 6, 7), "Queen’s Silver Jubilee"), ],
+        1981: [(date(1981, 7, 29), "Royal wedding"), ],
+        1999: [(date(1999, 12, 31), "New Year's Eve"), ],
+        2002: [(date(2002, 6, 3), "Queen’s Golden Jubilee"), ],
+        2011: [(date(2011, 4, 29), "Royal Wedding"), ],
+        2012: [(date(2012, 6, 5), "Queen’s Diamond Jubilee"), ],
+    }
 
-    def get_variable_days(self, year):
-        days = super(UnitedKingdom, self).get_variable_days(year)
-        days += [
-            Holiday(
-                date(year, 5, 1) + rd.relativedelta(weekday=rd.MO(1)),
+    def get_early_may_bank_holiday(self, year):
+        """
+        Return Early May bank holiday
+        """
+        # Special case in 2020, for the 75th anniversary of the end of WWII.
+        if year == 2020:
+            return Holiday(
+                date(year, 5, 8),
+                "Early May bank holiday (VE day)",
+                indication="VE day",
+            )
+        else:
+            return Holiday(
+                UnitedKingdom.get_nth_weekday_in_month(year, 5, MON),
```

Comment:
Here I think the use of relativedelta should be preferred.

Reply:
First, I'm on slightly shaky ground with relativedelta, but the more substantive point is that upstream does it this way. If you still think relativedelta is preferred, a hint would be appreciated. I'll wait for your response as you noted this in several places.